src/components/List/index.tsx

```diff
@@ -77,23 +108,21 @@ const StyledListSubItem = styled(ListItem)<ListItemProps>`
 export const StyledListItemText = styled(ListItemText)`
   span {
     font-family: ${({ theme }) => theme.fonts.fontFamily};
-    font-size: 0.76em;
+    font-size: 14px;
     font-weight: 600;
     line-height: 1.5;
-    letter-spacing: 1px;
-    color: ${({ theme }) => theme.colors.placeHolder};
-    text-transform: uppercase;
+    letter-spacing: 0.5px;
+    color: #162d45 !important;
```

Comment:
Why are we using a color not from our theme? Can we use the closest to it?

Reply:
Good catch, I added the variable to our theme and used it instead.